Client/Teztez/Core/Architecture/Observable.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  Observable.swift
+//  Teztez
+//
+//  Created by Adlet on 4/24/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import Foundation
+
+@propertyWrapper
+final class Observable<Value> {
```

Comment:
I thought you decided to use appropriate construct from [Apple's Combine](https://developer.apple.com/documentation/combine), I think [Published](https://developer.apple.com/documentation/combine/published) - is the thing.

Maybe I missed something and you decided not to use it, correct me if I am wrong here